<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Resume</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">HE, Zhouyi</div>
<div class="menu-item"><a href="index.html">About</a></div>
<div class="menu-item"><a href="bio.html" class="current">Resume</a></div>
<div class="menu-item"><a href="photography.html">Photography</a></div>
<div class="menu-item"><a href="https://space.bilibili.com/29220744/video">Videos</a></div>
<div class="menu-item"><a href="https://www.linkedin.com/in/%E5%91%A8%E6%98%93-%E8%B3%80-8620a019b/?locale=en_US">LinkedIn</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Resume</h1>
</div>
<h2>Education</h2>
<p><b><a href="https://hkust.edu.hk/home">Hong Kong University of Science and Technology</a> (HKUST)</b><br /></p>
<ul>
<li><p>M.Phil. in Chemistry (Theoretical Biophysics &amp; Computational Biology)&nbsp; &nbsp; &nbsp; &nbsp; 08/2019 - 06/2022</p>
</li>
</ul>
<p><b><a href="https://en.ustc.edu.cn">University of Science and Technology of China</a> (USTC)</b><br /></p>
<ul>
<li><p>B.Sc. in Chemistry (Chemical Physics) &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; ···&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;08/2015 - 06/2019</p>
</li>
</ul>
<h2>Research exprience</h2>
<p><b>Prof. <a href="http://haibinsu.people.ust.hk">Haibin Su</a>'s research group, department of chemistry, HKUST</b><br /></p>
<ul>
<li><p>Sequence-dependent Target searching and binding dynamics of CRISPR Cas9 &nbsp;  06/2021 - now<br />
PAM searching dynamics of CRISPR Cas9, is modelled by stochastic equations with the frame work of facilitated diffusion, namely the combination of 3D diffusion and 1D sliding. This work fills the gap of Cas9 target search dynamics and could be combined with sequence-dependent models to further understand the binding free energy landscape of other DNA-targeting proteins.<br />
<br />
Combined with protein sequence analysis, a conformation-dependent model is constructed to link the off-target effect and kinetic changes and to explain the sequence discrimination in target binding. This work provides a comprehensive understading of Cas binding dynamics thus sheds lights on Cas9 variants design and off-target rate minimization. </p>
</li>
</ul>
<ul>
<li><p>Evolution of CRISPR Cas9 systems in streptococcus genus &nbsp; &nbsp; &nbsp; &nbsp; 06/2020 - 05/2021<br />
Statistical coupling analysis is applied to retrieve sectors, which are the evolutionary units of 3-D structure that could reflect their evolutionary histories. Besides, A conformation-dependent model is proposed to link the mutations in Cas9 protein and kinetic changes. We found the relation of mutations in key residues and off-target effects, which also suggests that there exists a negative relation between accuracy and efficiency.</p>
</li>
</ul>
<ul>
<li><p>Dynamic Expedition of Leading Mutations in SARS-CoV-2 Spike Protein &nbsp; &nbsp; &nbsp; &nbsp; 05/2021 - 01/2022<br />
During the ongoing CoVID-19 epidemic, the continuous genomic evolution of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been generating new variants with enhanced transmissibility and immune escape. Being one key target of antibodies, mutations of the spike glycoprotein play a vital role in the trajectory of virus evasion. Here, we present a time-resolved statistical method, dynamic expedition of leading mutations (deLemus), to analyze the evolution dynamics of the spike protein. Together with analysis on single amino-acid polymorphism (SAP), we proposed one L-index to quantify the mutation strength of each amino acid for unravelling the mutation pattern of spike glycoprotein. The sites of interest (SOI) with high L-index hold great promise to detect the potential signal of emergent variants.</p>
</li>
</ul>
<p><b>Prof. <a href="http://staff.ustc.edu.cn/~xz58/">Xiao Zheng</a>'s research group, department of chemical physics, USTC</b><br /></p>
<ul>
<li><p>Unit Cell Consistency of Maximally Localized Wannier Functions &nbsp; &nbsp; &nbsp; &nbsp; 03/2019 - 06/2019<br />
Theoretical derivation and computation validation of unit cell consistency of MLWFs, which provided a theoretical foundation for further developments of material computation.</p>
</li>
</ul>
<p><b>Prof. <a href="https://small.hkust.edu.hk/Members.html">Jinqing Huang</a>'s research group, department of chemistry, HKUST</b><br /></p>
<ul>
<li><p>Chemical-Physical and Aggregation Properties of alpha-synuclein &nbsp; &nbsp; &nbsp; &nbsp; 07/2018 - 08/2018<br />
Structural characterization of toxic oligomers to properties of single-molecule alpha-synuclein using Optical Tweezers and Ramon Spectroscopy.</p>
</li>
</ul>
<p><b>Prof. <a href="http://staff.ustc.edu.cn/~junling/index.html">Junling Lu</a>'s research group, department of chemical physics, USTC</b><br /></p>
<ul>
<li><p>Design of Single-Atom Catalyst with Atomic Layer Deposition &nbsp; &nbsp; &nbsp; &nbsp; 03/2017 - 06/2018<br />
Synthesis of catalysts with Pt atomic layer deposition on g − C3N4; optimization of their performance in catalyzing hydrogen-releasing reactions by testing reaction activity and stability.</p>
</li>
</ul>
<h2>Publication</h2>
<ul>
<li><p><a href="https://doi.org/10.1088/2516-1075/ab5e5a">Yang, Xiaolong, Zhouyi He, and Xiao Zheng. &ldquo;Unit cell consistency of maximally localized Wannier functions.&rdquo; Electronic Structure 2.1 (2020): 014001.</a></p>
</li>
</ul>
<ul>
<li><p><a href="https://doi.org/10.1101/2021.12.29.474427">He, Zhouyi, et al. &ldquo;Dynamic Expedition of Leading Mutations in SARS-CoV-2 Spike Glycoproteins.&rdquo; bioRxiv (2021).</a> (Preprint, under review)</p>
</li>
</ul>
<h2>Knowledge &amp; Skills</h2>
<ul>
<li><p>Theoretical Chemical Physics: Physical Chemistry, Reaction Dynamics, Quantum Physics, Statistical mechanics, Stochastic Thermodynamics, etc.</p>
</li>
<li><p>Computational Chemistry: Monte Carlo, Molecular Dynamics, Density Functional Theory, etc.</p>
</li>
<li><p>Computational Biology: Bioinformatics, Statistical Coupling Analysis, Direct Coupling Analysis, etc.</p>
</li>
<li><p>Quantitative and Systems Biology: Growth model, Moran model, Quasispecies equation, Clonal Interference Evolutionary Game theory, etc.</p>
</li>
<li><p>Software/Coding: Python, MATLAB, Mathematica, C, Latex, Origin, Gaussian, Gromacs, Wannier 90, Pymol, BLAST, MSA tools, etc.</p>
</li>
</ul>
<h2>Honor &amp; Service</h2>
<ul>
<li><p>Postgraduate Studentship @HKUST, 2019-2022<br /></p>
</li>
<li><p>National Endeavor Scholarship @USTC  2016, 2018<br /></p>
</li>
<li><p>Outstanding student Scholarship @USTC 2016, 2018 <br /></p>
</li>
<li><p>Second Prize in 28th China Chemistry Olympic Competition 2014<br /></p>
</li>
<li><p>Championship and organizer in university basketball competition @USTC 2017<br /></p>
</li>
<li><p>Leader of teaching volunteers of aid education @Western Hunan 2015<br /></p>
</li>
</ul>
<h2>Hobby</h2>
<ul>
<li><p>Photography, Literature, Music, Movie, Hiking, Traveling, Basketball, etc.</p>
</li>
</ul>
<p>See more in my academic <a href="./resume.pdf">CV</a>.<br /></p>
<div id="footer">
<div id="footer-text">
Page generated 2022-11-16 21:25:52 CST, by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
